chapters/inheritance.tex

```diff
@@ -1270,4 +1270,4 @@ \section{Selective Model Extension}\label{selective-model-extension}
 \end{lstlisting}
 In this case the deselection removes all of the connect-statements.
 \end{example}
-\end{nonnormative}
\ No newline at end of file
+\end{nonnormative}
```

Comment:
I used "Edit this file" on github to modify it, and it also modified the white space at the end of the file. Not sure if this is crucial or how to revert that.

Reply:
It should be there.  Let's try this:
```suggestion
\end{nonnormative}

```